libexec/pyenv-init

```diff
@@ -40,6 +40,9 @@ done
 shell="$1"
 if [ -z "$shell" ]; then
   shell="$(ps -p "$PPID" -o 'args=' 2>/dev/null || true)"
+  if [ -z "$shell" ]; then
+    shell="$(ps -p $PPID 2>/dev/null | awk 'END {print $NF}' 2>/dev/null)"
```

Comment:
* Will terminate Pyenv if the invocation fail for whatever reason
* Will silently produce "COMMAND" if `ps` doesn't print the info for whatever reason (e.g. process is not found or cannot be opened).

Reply:
Alright. I see it. I'll hopefully be back on a linux distribution soon, instead of having to get used to windows. Thanks for taking your time to answer me, though.

When I'm back on Linux, I'll see if there are any open issues, I could address here. Overall I'm new to contributing to open source and this experience has been very pleasant. Keep up the good work.

Reply:
The docs recommend using https://github.com/pyenv-win/pyenv-win for Windows. It does installed "real" win32 Pythons.